cmake/pcre.cmake

```diff
@@ -1,12 +1,12 @@
-INCLUDE (CheckCSourceRuns)
 INCLUDE (ExternalProject)
 
 SET(WITH_PCRE "auto" CACHE STRING
    "Which pcre to use (possible values are 'bundled', 'system', or 'auto')")
 
 MACRO(BUNDLE_PCRE2)
   SET(dir "${CMAKE_BINARY_DIR}/extra/pcre2")
-  SET(PCRE_INCLUDES ${dir}/src/pcre2-build ${dir}/src/pcre2/src)
+  SET(PCRE_INCLUDE_DIRS ${dir}/src/pcre2-build ${dir}/src/pcre2/src)
```

Comment:
This renaming will break ColumnStore, best to flag this to that team.

Reply:
Turns out 10.6 (develop-6) was already out of date - https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/3304
develop-5 doesn't appear to be getting updated.

Appended commit for it d80ce3d905